packages/app/pages/Organization/cardStyle.scss

```diff
@@ -0,0 +1,52 @@
+/*- Typograpy and resets -*/
+.card {
+
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  border-style: solid;
+  border-width: thin;
+  border-radius: 4px;
+  background-color: white;
+  padding: 8px;
+  margin-top: 6px;
+  margin-left: 60px;
+  margin-right: 6px;
+  margin-bottom: 6px;
+  max-width: 75%;
+  flex-basis: 75%;
+  flex-grow: 1;
+}
+
+.name {
+  align-self: center;
+  justify-content: flex-start;
+}
+
+.details {
+  align-self: center;
+  margin-left: auto;
+
+}
+
+/*- Card container -*/
+.card-container {
+  position: relative;
+  z-index: 1;
+  margin: 32px auto;
+  width: 75%;
+  height: 420px;
+  perspective: 1000px;
```

Comment:
@StrikeNP , I know you were asking about this earlier.  This is being recognized as a syntax error

Reply:
Resolved this, this attribute wasn't used at all so it has been removed